modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -82,6 +82,10 @@ export default function _JournalPortlet({
 			}
 		};
 
+		actionInput.value = articleId
```

Comment:
I have a question here: this is also done in `handlePublishButtonClick`, which now I guess it wouldn't be necessary because it is set here in the component initialization. 
So, shouldn't we remove that from `handlePublishButtonClick`, as well as the else condition

```
if (!editingDefaultValues) {
...
else {
      actionInput.value = articleId
	  ? '/journal/update_article'
	  : '/journal/add_article';
}
```

and add it also where you did? Furthermore.... shouldn't be those values set on the JSP if they are fixed? (here I may be missing something...

Reply:
Hey @ambrinchaudhary ,

You are right. In this case that part is redundant, however the if block remains as its removing the required attribute from the ddm fields as well. I remove it as well as the test as I did not notice [LPD-28728: the default value of a structure is not deleted after the structure update](https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1078371//testReport/journal-web_journalArticle.spec/ts/LPD_28728__the_default_value_of_a_structure_is_not_deleted_after_the_structure_update)